flink-core/src/main/java/org/apache/flink/api/common/io/FileOutputFormat.java

```diff
@@ -305,6 +305,12 @@ public void tryCleanupOnError() {
 			this.fileCreated = false;
 			
 			try {
+				close();
+			} catch (IOException e) {
+				LOG.error("Could not properly close FileOutputFormat.", e);
```

Comment:
We can't check for that. We could expose the outputstream, but I couldn't find a way to check whether the stream was actually closed or not.